### ISSUE TITLE
getWeb3ClientVersion() returns null

### ISSUE THREAD

**Issue by KebAz** [web3j/web3j#956]
I'm trying to create an android Dapp by using the web3j library. I'm using connection to a local private Ethereum blockchain. When I'm trying to display the Client version (in my Android app) it returns null. This is the code:
```
static public Web3j web3j = null;
 static String clientUrl = "http://192.168.1.33:8501"; 
 Web3ClientVersion client = new Web3ClientVersion();
 web3j = Web3jFactory.build(new HttpService(clientUrl));
       try {
        client = web3j.web3ClientVersion()
                 .sendAsync()
                 .get();
    }  catch (InterruptedException e) {
            e.printStackTrace();
        } catch (ExecutionException e) {
            e.printStackTrace();
       }
 Log.e("client info: ", "Client version:  " + client.getWeb3ClientVersion()  + "\n");
```

The json-rpc api is enabled by using this command :
```
geth --datadir node1/ --syncmode full --port 30311 --rpc --rpccorsdomain "*" --rpcaddr 192.168.1.33 --rpcport 8501 --rpcapi personal,db,eth,net,web3,txpool,admin,miner --networkid 1818 -unlock 0x7f4a1b602d8ecf4439fc3e58d2e5a388730ee9fc --password node1/password.txt --mine
```

**Comment by iikirilov** [comment]
The code snippet is invalid. The second catch block is not closed.

@KebAz Is this still an issue?

**Comment by KebAz** [comment]
> The code snippet is invalid. The second catch block is not closed.
> 
> @KebAz Is this still an issue?

Hi @iikirilov, thanks for your reply .
No the catch block is closed, it was just a copy/past typo.
I still have the same issue.

**Comment by iikirilov** [comment]
@KebAz I need more information to reproduce this. This is not a web3j as the command does work correctly. Please could you provide information about your environment, mainly what type of node are you using (i.e. geth/parity/ganache...). Any other relevant information may be useful too.

**Comment by KebAz** [comment]
> @KebAz I need more information to reproduce this. This is not a web3j as the command does work correctly. Please could you provide information about your environment, mainly what type of node are you using (i.e. geth/parity/ganache...). Any other relevant information may be useful too.

Hi @iikirilov,

I'm working on a Windows (8.1) machine, I installed Geth (version: 1.7.0-stable).
I created a private blockchain with 2 nodes in the same machine. Then I created a mobile app by using android. So the communication with the Ethereum nodes will be done by using the web3j library.

For that I did these steps:
1- I created a bootnode, then I run it in a cmd: `bootnode -nodekey boot.key -verbosity 9 -addr :30310`
2- Then I run the first node by using this command: 
`geth --datadir node1/ --syncmode full --port 30311 --rpc --rpccorsdomain "*" --rpcaddr 192.168.1.33 --rpcport 8501 --rpcapi personal,db,eth,net,web3,txpool,admin,miner --bootnodes enode://dd06682752f3163955549994359c0035b7344114b5ece0042fd79a1173af2e91f2332dd85cec07ec430f9a26f1f1b40e8cac6ac72402dde23e6e3a8144aa6501@127.0.0.1:30310 --networkid 1818 -unlock 0x7f4a1b602d8ecf4439fc3e58d2e5a388730ee9fc --password node1/password.txt --mine`
3- Then the second one by using this command:
`geth --datadir node2/ --syncmode full --port 30312 --rpc --rpccorsdomain "*" --rpcaddr 192.168.1.33 --rpcport 8502 --rpcapi personal,db,eth,net,web3,txpool,admin,miner --bootnodes enode://dd06682752f3163955549994359c0035b7344114b5ece0042fd79a1173af2e91f2332dd85cec07ec430f9a26f1f1b40e8cac6ac72402dde23e6e3a8144aa6501@127.0.0.1:30310 --networkid 1818 -unlock 0xb1313811d2c2d2fc18a0d2126cd728d2d0246676 --password node2/password.txt --mine --ipcdisable console`
As a result of that : The blockchain is working correctly.

4- On the Android side: 
I'm runing my app in an external smartphone Samsung (S6).
To test the connection with the blockchain, I used a logCat method to display the client version (of the node to which my app is connecting):  
`Log.e("client` info: ", "Client version:  " + client.getWeb3ClientVersion()  + "\n");`
